test/error.ts

```diff
@@ -36,7 +36,7 @@ test('properties', withServer, async (t, server, got) => {
 test('catches dns errors', async t => {
 	const error = await t.throwsAsync<RequestError>(got('http://doesntexist', {retry: 0}));
 	t.truthy(error);
-	t.regex(error.message, /ENOTFOUND/);
+	t.regex(error.message, /(ENOTFOUND|EAI_AGAIN)/);
```

Comment:
Why this change?

Reply:
On Ubuntu 18.04 and 20.04 it was throwing that error, meaning a DNS Lookup Timeout.

I think this should be ok for this test.